packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -126,20 +126,12 @@ function InteractionLayer({
   function onPointerMove(event) {
     if (creating) {
       activeTool?.handlePointerMove?.(convertEvent(event), activeMark)
-    } else {
-      // this outputs the mouse coords when not creating (ig: guideline for Polygon)
-      activeTool?.handlePointerPosition?.(convertEvent(event), activeMark)
     }
   }
 
   function onFinish(event) {
-    if (event?.preventDefault) event.preventDefault()
+    event?.preventDefault?.()
     setCreating(false)
-    if (activeMark && !activeMark.isValid) {
-      activeTool.deleteMark(activeMark)
-      setActiveMark(undefined)
-      event?.stopPropagation()
-    }
```

Comment:
OK, the "if mark is invalid, delete it" logic has been moved to mark.finish(). 👌  (pt 1/2)